sycl/include/CL/sycl/queue.hpp

```diff
@@ -237,6 +237,7 @@ class __SYCL_EXPORT queue {
   template <info::queue param>
   typename info::param_traits<info::queue, param>::return_type get_info() const;
 
+private:
```

Comment:
If it doesn't break any test, I will be surprised.

Reply:
@bader this function is not on the spec, and it was introduced to be used internally. I don't think there're any tests for this function.

Reply:
This method is only used internally.

Reply:
This is a violation of the contribution process. See https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#tests-development: 
> Every product change should be accompanied with corresponding test modification (adding new test(s), extending, removing or modifying existing test(s)).

Please, make sure that code contributed to the project is covered by some tests.

Reply:
> This method is only used internally.

You can't claim that for public API. It's used in SYCL-CTS - https://github.com/KhronosGroup/SYCL-CTS/pull/287#discussion_r814092413.

Reply:
http://ci.llvm.intel.com:8010/#builders/18/builds/21660 - that's what I expected. Although Linux testing results are "surprising".

Reply:
On the contrary, I didn't expect failure on Windows platform. I didn't know they mangle access modifier in symbol name.